File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void Loop(int Width);

int main(void)
{
   int Height = get_int("Enter Height");
   if (Height <= 0 || Height > 8)
   {
        Height = get_int("Enter Height");
   }
   int Width = 1;
   for (int i = 0; i < Height; i++ )
   {
        for (int l = 0; l < (Height-Width); l++ )
            printf(" ");
        Loop(Width);
        printf("  ");
        Loop(Width);
    Width = Width + 1;
    printf("\n");
   }
}

void Loop(int Width)
{
    for (int j = 0; j < Width; j++)
    {
        printf("#");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

def loop(i):
    for j in range(i):
        print("#", end="")

height = get_int("Height ")
while height > 8 or height < 1:
    height = get_int("Height ")
width = 1
for i in range(height):
    for j in range(height - width):
        print(" ", end="")
    loop(width)
    print("  ", end="")
    loop(width)
    print()
    width += 1


# TODO